server/db-operations.js

```diff
@@ -16,37 +16,29 @@ var _prepForDb = function (cocktail) {
 
 module.exports = {
 
-  createCocktail: function (cocktailBody) {
-    var cocktail = new Cocktail(_prepForDb(cocktailBody));
+  createCocktail (cocktailBody) {
```

Comment:
Note that all these functions are using the object property shorthand, too: this is the equivalent of `createCocktail: (cocktailBody) => {...`, which is the equivalent of `createCocktail: function (cocktailBody) {...`